src/types/User.ts

```diff
@@ -5,6 +5,8 @@ export interface Experience {
   column?: number;
 }
 
+export type Mapping = { name: string; path: string };
```

Comment:
Couldn’t we just do this as a standard object? Where we use the key for name, and the oath is the value. Rather than needing to ‘double up’ like this.

then we could simplify implementation where it’s used as well to just be Object.entries(mapping).map(([name, path]) => [name, get(foo, path)]).filter(where value not defined) etc

There might even be a direct .mapValues or similar for the entries, can’t remember if that’s standard or just a lib I used once.

Reply:
Partially done in https://github.com/sparkletown/sparkle/pull/1667/commits/15dfaa4a6348a87a57db3386e982e12358de521c

I didn't find any value in removing `reduce` from there. It does filtering + mapping per single go-through. Even though it's just 5-6 fields, I don't think mapping + filtering adds any cognitive value

Reply:
Since it’s already written it’s somewhat moot, but it’s less overall boilerplate code required to implement, which is why I suggested it. Simplifies the code down dramatically. Tradeoff is that filter will run through it a second time

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/2d4f2617700b9bb64a4edaab5c88b7bcb223540c + https://github.com/sparkletown/sparkle/pull/1667/commits/c77cd255fb597da0dd2611bc66089e77dbe0e59f